src/generateArray.js

```diff
@@ -1,7 +1,8 @@
-import { map, times } from 'lodash'
+import { map } from 'lodash';
 // const utils = require('@utils')
-const { generateArrWithId } = require('./utils');
+import { generateArrWithId } from './utils';
 
+const generateArrWithId = generateArrWithId;
```

Comment:
Why need this assignment ?

Reply:
nobody knows :) but looks like it was removed